pkgs/applications/misc/web-media-controller/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, glib, pcre, json-glib, gobject-introspection }:
+
+stdenv.mkDerivation rec {
+  pname = "wmc-mpris";
+  name = "${pname}-unstable";
+  version = "2019-04-12";
+
+  src = fetchFromGitHub {
+    owner = "f1u77y";
+    repo = pname;
+    rev = "fcfb1d8f708b8e69bc2ab8b9bc0e2605d9ace585";
+    sha256 = "0plknl1r3z58sl68s60z4cxlwk7z5ld09lza74x5m8knhr94vwy9";
+  };
+
+  nativeBuildInputs = [ cmake pkgconfig ];
+  buildInputs = [ glib pcre json-glib gobject-introspection ];
```

Comment:
Where is `gobject-introspection` needed and how?

Reply:
Correct, I thought it is needed because this package connects to D-Bus but never mind. Fixed that and the commit message in the last push.